lib/packwerk/configuration.rb

```diff
@@ -22,7 +22,10 @@ def from_path(path = Dir.pwd)
       private
 
       def from_packwerk_config(path)
-        new(YAML.load_file(path), config_path: path)
+        new(
+          YAML.load_file(path) || {},
```

Comment:
Can you move this line of code into a variable dedicated variable? That way the instantiation becomes a bit easier to read.

Reply:
Ah, I actually purposefully avoided it to minimize allocations, but I can certainly add one. :+1:

Reply:
Personally, I find it more difficult to read with a variable in there:

```ruby
yaml_config = YAML.load_file(path) || {}
new(yaml_config, config_path: path)
```